README.md

```diff
@@ -113,7 +170,7 @@ Description:
     Create react on rails files for install generator.
 ```
 
-We have a repo showing the results of running the generator with various combinations of options, each combination on its own branch: [Generator Results](https://github.com/shakacode/react_on_rails-generator-results-pre-0/pulls).
+We have a repo showing the results of running the generator with various combinations of options, each combination on its own branch: [Generator Results](https://github.com/shakacode/react_on_rails-generator-results-1-0-0/pulls).
```

Comment:
The pull requests demonstrate the change from the generator given different options, comparing both from a brand new Rails app, and between using different options:

@robwise convert to links:

1. Basic (provide generator command)
2. Basic with Server Rendering (options)
3. Redux
4. Redux with Server Rendering
5. Basic to Redux
6. Basic to Basic Server Rendering
7. Redux to Redux Server rendering